client/express-checkout/utils/checkPaymentMethodIsAvailable.js

```diff
@@ -15,11 +15,11 @@ import { getUPEConfig } from 'wcpay/utils/checkout';
 
 export const checkPaymentMethodIsAvailable = memoize(
 	( paymentMethod, cart, resolve ) => {
-		const root = ReactDOM.createRoot(
-			document.getElementById(
-				`express-checkout-check-availability-container-${ paymentMethod }`
-			)
-		);
+		// Create the DIV container on the fly
+		const containerlEl = document.createElement( 'div' );
+		document.querySelector( 'body' ).appendChild( containerlEl );
```

Comment:
If we're going to create the div like this we may want to add some attributes to it that make it invisible, especially for screen readers.

Probably something like

```js
containerEl.style.opacity = 0;
containerEl.style.height = 0;
conatinerEl.ariaHidden = true;
```

That's roughly similar to the previous approach of setting the opacity and height to 0, although that approach does not set the aria-hidden attribute.

Reply:
Yes I agree, CSS Styles were added here [f62431b](https://github.com/Automattic/woocommerce-payments/pull/9420/commits/f62431bf4c8a98b0a9431681241f4a120c0da7aa)